src/NexusMods.App.UI/Controls/TopBar/TopBarViewModel.cs

```diff
@@ -49,6 +57,12 @@ public TopBarViewModel(ILogger<TopBarViewModel> logger, ILoginManager loginManag
                 .OnUI()
                 .SubscribeWithErrorLogging(logger, x => Avatar = x)
                 .DisposeWith(d);
+
+            workspaceController.WhenAnyValue(controller => controller.ActiveWorkspace!.Title)
+                .Select(title => title.ToUpperInvariant())
+                .OnUI()
+                .BindTo(this, vm => vm.ActiveWorkspaceTitle)
+                .DisposeWith(d);
```

Comment:
Changes to `ActiveWorkspace` are guaranteed to happen on the UI thread, the `OnUI` is redundant.